Packs/USTA/Integrations/USTA/USTA.yml

```diff
@@ -0,0 +1,170 @@
+    name: usta-get-malware-hashs
+  - arguments:
+    - auto: PREDEFINED
+      description: Filtering by status
+      name: status
+      predefined:
+      - open
+      - close
+      - in_progress
+      - out_of_scope
+      - passive
+    - description: Filtering by source(URL)
+      name: source
+    - description: Paginiation
+      name: page
+    description: You can get phishing sites with this command
+    name: usta-get-phishing-sites
+  - arguments:
+    - description: Staring Date
+      name: start
+    - description: End Date
+      name: end
+    description: With the Identity Leak API, you can access the hashed version of
+      the credentials added to the platform.SHA256(MD5(Identity_Number))
+    name: usta-get-identity-leaks
+  - arguments:
+    - description: Filtering by username
+      name: username
+    - description: Filtering by password
+      name: password
+    - auto: PREDEFINED
+      description: 'It allows to filter the stolen customer accounts detected according
+        to the source.Available values : malware, phishing_site, data_leak, clients'
+      name: source
+      predefined:
+      - malware
+      - phishing_site
+      - data_leak
+      - clients
+    - description: Starting Date
+      name: start
+    - description: End Date
+      name: end
+    description: You can access stolen customer accounts via Stolen-Client-accounts
+      API.
+    name: usta-get-stolen-client-accounts
+  - arguments:
+    - description: Search with domain name
+      name: domain
+      required: true
+    description: If you want to get more detailed information about malicious domain
+      names, you can use this command.
+    name: usta-get-domain
+  - arguments:
+    - description: Search with IP Address
+      name: ip_address
+    description: If you want to get more detailed information about specific IP Address,
+      you can use this command.
+    name: usta-get-ip-address
+  - arguments:
+    - description: 'URL Value. Example: http://www.google3.com'
+      name: address
+      required: true
+    description: You can search about the accuracy of the urls referring to your company's
+      websites.
+    name: usta-send-referrer-url
+  - arguments:
+    - description: 'Search with this identity number. You can search all identity
+        number with "," '
+      name: identity_number
+      required: true
+    description: 'With this command, you can search specific identity number that
+      hashed in leaks '
+    name: usta-search-specific-identity-leaks
+  - arguments:
+    - description: Incident ID
+      name: id
+      required: true
+    description: You can close the notifications in the status of "In Progress" or
+      "Open", which are currently opened to your institution, via API.
+    name: usta-close-incident
+  dockerimage: demisto/python3:3.9.2.17246
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+fromversion: 6.0.0
+tests:
```

Comment:
I don't see any outputs for any of the commands. 
Here is some documentation about it: https://xsoar.pan.dev/docs/integrations/context-and-outputs
When you provide outputs to the commands you can choose them directly in a playbook and don't need to type it manually, this is highly recommended.

Reply:
Resolved.